tests/posts_tests/test_posts_views.py

```diff
@@ -2,43 +2,61 @@
 from pytest_django.asserts import assertTemplateUsed
 from django import urls
 from django.contrib.auth.models import User
-
 from conftest import EMPTY_COMMENT, LONG_COMMENT, VALID_COMMENT
```

Comment:
Not related to this PR, but the import should be `from tests.conftest` and not `from conftest`

Reply:
Are you sure? when I try it I get:
![image](https://user-images.githubusercontent.com/90772515/168467607-5014ee97-4029-44b9-a036-24caf7ec8186.png)

Reply:
Yes, now I see, my environment was messy, I encountered a faulty error, please ignore this comment